automation/src/test/scala/org/broadinstitute/dsde/test/api/SnapshotAPISpec.scala

```diff
@@ -110,8 +99,60 @@ class SnapshotAPISpec extends AnyFreeSpecLike with Matchers with BeforeAndAfterA
       }
     }
 
+    "should report the same tables/columns via metadata API as TDR reports" taggedAs(Tags.AlphaTest, Tags.ExcludeInFiab) in {
+      val numSnapshotsToVerify = 2
+
+      // as of this writing, hermione.owner is the user with access to snapshots
+      val owner = UserPool.userConfig.Owners.getUserCredential("hermione")
+
+      implicit val ownerAuthToken: AuthToken = owner.makeAuthToken()
+
+      // get N snapshots from TDR
+      val drSnapshots = listDataRepoSnapshots(numSnapshotsToVerify, owner)(ownerAuthToken)
+
+      withCleanBillingProject(owner) { projectName =>
+        withWorkspace(projectName, s"${UUID.randomUUID().toString}-snapshot references") { workspaceName =>
+          // loop through each snapshot, and:
+          drSnapshots.getItems.asScala.foreach { snapSummary =>
+            info(s"for snapshot ${snapSummary.getId}: ${snapSummary.getName}")
+
+            val referenceName = s"refname-${snapSummary.getId}".replaceAll("-", "_")
```

Comment:
There may be an obvious answer to this - but why do you we need to replace `-` with `_`? Worth adding a brief comment/explanation?

Reply:
I added a code comment to explain this!